src/main/java/net/dv8tion/jda/entities/TextChannel.java

```diff
@@ -34,4 +36,14 @@
      *      The String needed to mention this Channel
      */
     String getAsMention();
+    
+    /**
+     * Bulk deletes a list of messages. Must not be more than 100 messages at a time.
+     * You must have permission to delete a message. You may always delete your own messages. In the event that you do not have permission to delete a message, the message will be skipped.
```

Comment:
This isn't quite correct, you need MESSAGE_MANAGE to bulk delete anyway so all messages in the list will be deleted..